speech/api/speech_async_rest.py

```diff
@@ -57,8 +57,12 @@ def main(speech_file):
     service_request = service.speech().asyncrecognize(
         body={
             'config': {
-                'encoding': 'LINEAR16',
-                'sampleRate': 16000
+                # There are a bunch of config options you can specify. See
+                # https://cloud.google.com/speech/reference/rest/Shared.Types/RecognitionConfig  # noqa
```

Comment:
It's 'cuz it's a long line - if I don't put the # noqa comment, the linter will complain that the line exceeds 79 characters in length - which normally we don't want, but you can't really break up a url in a comment..